app/client/src/utils/getName.ts

```diff
@@ -0,0 +1,14 @@
+export const getName = (name: string) => {
```

Comment:
Real nice,
Could we have this here too?
https://github.com/shramee/starklings-cairo1/blob/main/app/client/src/components/pages/Workspace/Workspace.tsx#L191